python/lsst/dax/imgserv/cli_v1.py

```diff
@@ -42,8 +42,24 @@
 from lsst.dax.imgserv.hashutil import Hasher
 from lsst.dax.imgserv.jsonutil import flatten_json
 
+try:
+    from ConfigParser import RawConfigParser, NoSectionError
+except ImportError:
+    from configparser import RawConfigParser, NoSectionError
+
 ROOT = os.path.abspath(os.path.dirname(__file__))
 
+defaults_file = os.environ.get("WEBSERV_CONFIG", "~/.lsst/webserv.ini")
+
+# Initialize configuration
+webserv_parser = RawConfigParser()
+webserv_parser.optionxform = str
+
+with open(os.path.expanduser(defaults_file)) as cfg:
+    webserv_parser.readfp(cfg, defaults_file)
+
+webserv_config = dict(webserv_parser.items("webserv"))
+imgserv_meta_url = webserv_config.get("dax.imgserv.meta.url")
```

Comment:
Is this related to the tests?  I'm not sure I get why this is changing?  Or perhaps this is something else that was different?

Reply:
Ah gotcha, makes sense.